src/applications/vaos/components/review/ReviewDirectScheduleInfo.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { getTypeOfCare } from '../../utils/selectors';
+import ReasonForAppointmentSection from './ReasonForAppointmentSection';
+import ContactDetailSection from './ContactDetailSection';
+import AppointmentDate from './AppointmentDate';
+
+export default function ReviewDirectScheduleInfo({ data, facility, clinic }) {
+  return (
+    <div>
+      <h1 className="vads-u-font-size--h2">
+        Review your appointment
+        <br />
```

Comment:
You can remove this break, it's fine to let it wrap where it needs to.

Reply:
Done